pep-0544.txt

```diff
@@ -1377,7 +1377,8 @@ Support adapters and adaptation
 -------------------------------
 
 Adaptation was proposed by :pep:`246` (rejected) and is supported by
-``zope.interface``, see https://docs.zope.org/zope.interface/adapter.html.
+``zope.interface``, see `the Zope documentation on adapter registries
+<https://web.archive.org/web/20160802080957/https://docs.zope.org/zope.interface/adapter.html>`_.
```

Comment:
Using an archive link is probably a good idea here, but if the current version of the document is of interest, it is at https://github.com/zopefoundation/zope.interface/blob/master/docs/adapter.rst

Reply:
Thanks for the suggestion, but I did mention the deployed version of that page in the PR description:

> I also found <https://zopeinterface.readthedocs.io/en/latest/adapter.html>, which contains very similar content